sp800-63a/sec6_dc.md

```diff
@@ -2,64 +2,46 @@
 
-1. A _claimant_ obtains a secondary, or derived authenticator, for use only within the limits and authorizations of the primary authenticator.  For example, this could be a derived PIV credential in federal use cases, or a temporary authenticator. See [Section 6.1](#dc) for more details.
-2. An _applicant_ seeks to obtain another primary authenticator from a CSP that did not issue the original authenticator. For example, if an applicant wants to switch CSPs and/or have another authenticator at a new CSP for other uses (e.g. basic browsing vs. financial). See [Section 6.2](#prior) for more details.
+The definition of derived in this section does *not* infer that an authenticator is cryptographically tied to a primary authenticator, for example deriving a key from another key.  Rather, the derived authenticator is issued and bound to an identity that has already been identity proofed.  To prove that identity proofing has occurred, the individual demonstrates possession and authentication of an authenticator bound to the original proofed identity record (i.e the credential). 
 
-## <a name="dc"></a> 6.1. Issuance Requirements for Derived Authenticators
+There are two specific use cases to consider leveraging past proofing of an individual:
 
-This section applies when the secondary authenticator is directly tied to the authorizations of having, and proving possession of, a primary authenticator, typically issued by the same CSP.
+1. A _claimant_ obtains a secondary, or derived authenticator, for use only within the limits and authorizations of the primary authenticator.  For example, this could be a derived PIV credential in federal use cases, or a temporary authenticator. *This section covers this use case.*
+2. An _applicant_ seeks to obtain an authenticator, bound to a proofed identity, from a CSP that did not proof the individual or issue the original authenticator. For example, if an applicant wants to switch CSPs and/or have another authenticator at a new CSP for other uses (e.g. basic browsing vs. financial). *This use case is covered by allowable identity evidence in [Section 5.2](#validate).*
 
-### 6.1.1. General Requirements
+For the first use case above, the management of derived authenticators is similar to that described in [Section 6.1.2. Post-Enrollment Binding](https://pages.nist.gov/800-63-3/sp800-63b.html#post-enroll-bind); however, derived authenticators considered in this section are only issued to a subject that is authorized to have a primary authenticator.  Once the primary authenticator has been revoked, it is reasonable that all derived will also be revoked.
```

Comment:
"it is reasonable that all derived will also be revoked" -> "all authenticators derived from that primary authenticator SHALL be revoked."